### DISCUSSION TITLE
Korean translation

### DISCUSSION THREAD

**Discussion by chanjin65** [jamulussoftware/jamulus#2653] [General]
How can I contribute for the Korean translation?

**Comment by ann0see** [comment]
Hi! Big thanks for your contribution!

You can see the translation process in this file: https://github.com/jamulussoftware/jamulus/blob/master/docs/TRANSLATING.md

Feel free to ask one of the existing translators (e.g henk, ignotus, BLumina)

**Reply by pljones** [reply]
👍 on the thanks - it's always appreciated!
